index.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+require_relative 'lib/game'
+
+def render(board)
+  puts '   +----+----+----+----+----+----+----+----+'
+  board.positions.each_index do |row|
+    print "#{row}  "
+    board.positions[row].each_index do |column|
+      if board.positions[row][column].nil?
+        print '|    '
+      else
+        print "| #{board.positions[row][column]}  "
+      end
+    end
+    puts '|'
+    puts '   +----+----+----+----+----+----+----+----+'
+  end
+
+  print '   '
+  (0..7).each { |letter| print "  #{letter}  " }
+  puts ''
+end
+
+def input_figure(game)
+  puts 'Please enter the figure you want to move'
+  figure = gets.chomp.downcase
+  until game.valid_figure?(figure)
+    puts 'Please enter a valid figure'
+    puts 'You can choose from pawn, rook, bishop, knight, queen and king'
+    figure = gets.chomp.downcase
+  end
+  figure
+end
+
+def input_num
+  num = gets.chomp.to_i
+  until (0..7).include?(num)
+    puts 'Please enter a valid number, it has to be between 0 and 7'
```

Comment:
Chess board is traditionally marked from a1 to h8, you should definitely be using that system.